### PR TITLE
Add version 0.2 to jib-gradle task

### LINKED ISSUE #584: Migrate jib-* outputs resource to param + result
The current `jib-*` tasks are still using `PipelineResource`. They should be updated (new version) to use `params` (for image) and `results` (for the generated image digest)

/kind feature

### PR DESCRIPTION
**Pull request by rudeigerc**
Signed-off-by: rudeigerc <rudeigerc@gmail.com>

# Changes

Closes #584 

Add version 0.2 to `jib-gradle` task.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of task changed or new task added)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [x] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [x] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [x] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [x] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [x] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [x] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

File: task/jib-gradle/0.2/tests/run.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: jib-gradle-test-pipeline-run
+spec:
+  pipelineRef:
+    name: jib-gradle-test-pipeline
+  workspaces:
+  - name: shared-workspace
+    persistentvolumeclaim:
```
Comment (vinamra28) [inline task/jib-gradle/0.2/tests/run.yaml:64]: nit:
```suggestion
    persistentVolumeClaim:
```

**Comment by chmouel** [comment]
/lgtm

Looks good to me, thanks! 👍

File: task/jib-gradle/0.2/tests/run.yaml (hunk 1 above)
Comment (rudeigerc) [reply task/jib-gradle/0.2/tests/run.yaml]: Fixed.

**Review by vinamra28 — APPROVED** [review]
/meow

**Comment by chmouel** [comment]
/lgtm
/approve

🚀

**Pull request merged**